teams_accounts.go

```diff
@@ -112,6 +140,26 @@ func (api *API) TeamsAccountConfiguration(ctx context.Context, accountID string)
 	return teamsConfigResponse.Result, nil
 }
 
+// TeamsAccountLoggingConfiguration returns teams account logging configuration.
+//
+// API reference: TBA.
```

Comment:
i see GATE-2311 is targeting getting this publicly documented. should we wait until that lands for the public stability?

Reply:
confirmed the team is fine with this endpoint being used and will update the documentation following these changes.